libkbfs/kbfs_ops_test.go

```diff
@@ -3430,3 +3430,89 @@ func TestDirtyPathsAfterRemoveDir(t *testing.T) {
 	_, err = config.BlockCache().Get(ptrC)
 	require.NotNil(t, err)
 }
+
+func TestKBFSOpsBasicTeamTLF(t *testing.T) {
+	var u1, u2, u3 libkb.NormalizedUsername = "u1", "u2", "u3"
+	config1, uid1, ctx, cancel := kbfsOpsInitNoMocks(t, u1, u2, u3)
+	defer kbfsTestShutdownNoMocks(t, config1, ctx, cancel)
+
+	config2 := ConfigAsUser(config1, u2)
+	defer CheckConfigAndShutdown(ctx, t, config2)
+	session2, err := config2.KBPKI().GetCurrentSession(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+	uid2 := session2.UID
+
+	config3 := ConfigAsUser(config1, u3)
+	defer CheckConfigAndShutdown(ctx, t, config3)
+	session3, err := config3.KBPKI().GetCurrentSession(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+	uid3 := session3.UID
+
+	// These are deterministic, and should add the same TeamInfos for
+	// both user configs.
```

Comment:
Happy to `t.Log` them, but don't the line numbers of the `require` error messages already say exactly where the failure happened?  I don't think I've ever found myself wishing for more logging within a test function.